src/main/java/seedu/address/model/person/ClassId.java

```diff
@@ -32,8 +33,8 @@ public ClassId(String classId) {
     /**
      * Returns true if a given string is a valid grade.
      */
-    public static boolean isValidClassId(String test) {
-        return test.matches(VALIDATION_REGEX);
+    public static boolean isValidClassId(String classId) {
+        return classId.matches(VALIDATION_REGEX) && classId.startsWith("B") && classId.length() == 3;
```

Comment:
I realised we already have `VALIDATION_REGEX` which checks for an alphabet with 2 numbers currently. Should we use that instead? I would be fine with both methods.

Reply:
You could try something like `public static final String VALIDATION_REGEX = "[B][0-9]{2}";`

Then we can remove the `startsWith()` check and `length()` check. Lmk if it works!

Reply:
updated :)